content/doc/book/system-administration/viewing-logs.adoc

```diff
@@ -33,22 +33,30 @@ run `systemctl edit jenkins` and add the following:
 Environment="JENKINS_LOG=%L/jenkins/jenkins.log"
 ----
 
-=== Windows
+=== Windows (msi)
 
 By default logs should be at `%JENKINS_HOME%/jenkins.out` and
 `%JENKINS_HOME%/jenkins.err`, unless customized in
 `%JENKINS_HOME%/jenkins.xml`
 
-=== Mac OS X
+=== Mac OS 
 
 Log files should be at `+/var/log/jenkins/jenkins.log+`, unless
 customized in `org.jenkins-ci.plist`.
```

Comment:
```suggestion
Log files should be at `+/var/log/jenkins/jenkins.log+`, unless customized in `org.jenkins-ci.plist`.
```
one sentence per line makes it easier for reviewers to read over. I know the entire document is not like this, but if we make this text follow that format, the rest can be updated at any time.